### DISCUSSION TITLE
Anyone got this working with EKHBH008?

### DISCUSSION THREAD

**Discussion by hahalj** [raomin/ESPAltherma#378] [Q&A] [answered]
Has anyone got this working with the Altherma 8 - EKHBH008?
On the PCB I find the X10A connector, does anyone know what type it is, JST EH, XH or anything else?
![5004206](https://github.com/raomin/ESPAltherma/assets/4930174/c1fbaa02-58cb-441f-90a3-ed156b1df543)

**Comment by mdr-github** [comment]
The answer is right here in the project description: [https://github.com/raomin/ESPAltherma#8-pin-x10a-connection](https://github.com/raomin/ESPAltherma#8-pin-x10a-connection)

**Reply by hahalj** [reply]
I might be blind, but I can't see the connector type described there. Just which pins to use.

**Reply by mdr-github** [reply, marked as answer]
In various places the documentation mentions this is an JST EH connector. But it's much easier to use Dupont wires.

**Reply by hahalj** [reply]
ok, thanks
